types/stylelint/index.d.ts

```diff
@@ -205,6 +217,8 @@ declare module 'stylelint' {
 
 		export type GetLintSourceOptions = GetPostcssOptions & {
 			existingPostcssResult?: PostCSS.Result;
+			cache?: boolean;
+			fileCache?: FileCache;
```

Comment:
[suggest] How about adding `fileCache` to the `InternalApi` type like this?

```diff
export type InternalApi = {
	_options: LinterOptions & { cwd: string };
	_extendExplorer: ReturnType<typeof cosmiconfig>;
	_specifiedConfigCache: Map<Config, Promise<CosmiconfigResult>>;
	_postcssResultCache: Map<string, PostCSS.Result>;
+	_fileCache: FileCache,
```

```diff
function createStylelint(options = {}) {
...
+	stylelint._fileCache = new FileCache(stylelint._options.cacheLocation, stylelint._options.cwd);
```

Because `InternalApi` has the cache properties of `_specifiedConfigCache` and `_postcssResultCache` already.

Reply:
I moved the `fileCache` to `InternalApi`. [`ca2fb53`](https://github.com/stylelint/stylelint/pull/6356/commits/ca2fb534ba93e2dba40092945e6f56758c99a97e#diff-7ee3bec822a60d59e7dfdabf3fdf562eb48971e3edfad7687e2f5e8c998e2e89)